fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanAccountDomainServiceJpa.java

```diff
@@ -292,7 +290,7 @@ private LoanBusinessEvent getLoanRepaymentTypeBusinessEvent(LoanTransactionType
         } else if (isRecoveryRepayment) {
             repaymentEvent = new LoanTransactionRecoveryPaymentPreBusinessEvent(loan);
         } else if (repaymentTransactionType.isDownPayment()) {
-            repaymentEvent = new LoanTransactionDownPaymentPreBusinessEvent(loan);
+            repaymentEvent = new LoanTransactionMakeRepaymentPreBusinessEvent(loan);
```

Comment:
Are you sure? we dont wanna trigger downpayment business event for downpayment transaction?